tools/base/fpgainfo/bmcdata.c

```diff
@@ -63,26 +64,26 @@ fpga_result get_metrics(fpga_token token,
         uint64_t j = 0;
         switch (inquiry) {
         case ALL:
```

Comment:
ALL is too generic of a name.  We should add a prefix.

Reply:
fixed